src/App.js

```diff
@@ -67,14 +50,12 @@ class App extends Component {
 }
 
 const mapStateToProps = state => ({
-  posts: state.posts,
+  //posts: state.posts,
 });
 
 const mapDispatchToProps = dispatch => ({
-  loadPosts: () => dispatch(loadPosts()),
-  createPost: post => dispatch(createPost(post)),
-  editPost: post => dispatch(editPost(post)),
-  deletePost: id => dispatch(deletePost(id)),
+  selectEmptyPost: id => dispatch(selectPost()),
```

Comment:
`createPost` would be a more clear name. Function name should express intention, not implementation details.